apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.GenerateClient/Program.cs

```diff
@@ -275,7 +278,7 @@ public override SyntaxNode VisitMethodDeclaration(MethodDeclarationSyntax node)
                 node = node.WithBodySafe(
                     node.Invoke(
                         IdentifierName(requestParameterSyntax.Identifier),
-                        IdentifierName(nameof(CallSettings)).Member(nameof(CallSettings.FromCancellationToken)).Invoke(IdentifierName(CancellationTokenParameterName))));
+                        IdentifierName($"gaxgrpc::{nameof(CallSettings)}").Member(Identifier(nameof(CallSettings.FromCancellationToken))).Invoke(IdentifierName(CancellationTokenParameterName))));
```

Comment:
`IdentifierName(...)` should just be used to create simple names. With something like this, you should really create an AliasQualifiedNameSyntax. Use `ParseName(...)` here instead.